cmdAdd.go

```diff
@@ -30,6 +31,13 @@ func actionCmdAdd(cmd *cobra.Command, args []string) {
 	}
 
 	var err error
+	if len(cfg.Add.Profile) == 0 {
+		cfg.Add.Profile, err = readStdinLine("AWS Profile: ")
```

Comment:
This should be marked optional as it is not required for the vast majority of applications.